### PR TITLE
Add Eclipse Temurin

### PR DESCRIPTION
**Pull request by eddumelendez**
See https://blog.adoptopenjdk.net/2020/12/eclipse-adoptium-projects-ready-for-community-review/

### CODE REVIEW

File: app/controllers/JavaListController.scala [hunk 1]
```diff
@@ -73,6 +73,7 @@ class JavaListController @Inject()(versionRepo: VersionsRepository) extends Cont
     "open" -> "Java.net",
     "mandrel" -> "Mandrel",
     "sapmchn" -> "SAP",
+    "tem" -> "Eclipse Tem.",
```
Comment (marc0der) [inline app/controllers/JavaListController.scala:76]: Just a thought, but how about we just make it `Temurin`?

**Review by marc0der — APPROVED** [review]

**Comment by marc0der** [comment]
@eddumelendez Is this PR still relevant? If not, can it be closed?

**Comment by eddumelendez** [comment]
the next release will be under https://adoptium.net/ so make sense to add it.

**Comment by marc0der** [comment]
Okay, let's get the conflicts resolved and the checks green then happy to approve.

**Comment by eddumelendez** [comment]
@marc0der I have updated the PR. I would prefer to use `Eclipse Temurin` to be more precise but I think we will have an issue with the vendor length's name, right?

**Comment by marc0der** [comment]
Let's have a chat to discuss what the name length entails and we can make it happen.

**Review by marc0der — APPROVED** [review]
LGTM!

**Pull request merged**